server/bundler/loader.js

```diff
@@ -44,10 +53,21 @@ function getSectionsModule( sections ) {
 			'	load: function() {',
 			'		' + sectionLoaders,
 			'	}',
-			'};'
+			'};',
+			'\n',
+			'function loadCSS( cssUrls, context ) {',
+			'	var languageFromParams = getLanguage( context.params.lang )',
```

Comment:
`context.params.lang` is set _later_ than this call.
@scruffian @tug any idea where is it set? How can we get it here?

Reply:
It seems to be set when I go to http://calypso.localhost:3000/start/account/user/he

Reply:
it is _now_ after the latest changes... =)